content/design/line-styles/index.html

```diff
@@ -2,65 +2,136 @@
+      </div>
+      <div class="definition-code">
+        <div class="bb-field-label">Code</div>
+        <code>@include bb-border(row)</code>
+      </div>
+      <div class="definition-styling">
+        <div class="bb-field-label">Styling</div>
+        <div>1px dotted #cdcfd2</div>
+      </div>
+    </div>
+  </div>
+  
+<h2 class="bb-section-heading" id="corners">Corners</h2>
 
-<h1 class="bb-page-heading">Line styles</h1>
+<p>In general, the style of corners should be handled by the component. If you need to directly specify the corners of an element, the rule of thumb is to use rounded corners for interactive elements and square corners for containers.</p>
 
-<p>{{ stache.config.product_name_short }} uses various line styles to bound, divide, and highlight content. The lines are designed to maintain visual hierarchy regardless of how complex pages become or how many nested containers they include.</p>
-<p>As with other types of formatting or decoration in {{ stache.config.product_name_short }}, use lines minimally. Remember that white space and positioning can group or highlight content without extra subdivisions or boundaries.</p>
-<p>{{ stache.config.product_name_short }} uses 6 line styles that you can apply with a <a href="https://github.com/blackbaud/skyux/blob/master/scss/_mixins.scss">SASS mixin</a>.</p>
-<ul>
-    <li>A light separator to organize and divide blocks of content within a container</li>
-    <li>A dark separator to indicate borders and boundaries for components</li>
-    <li>4 prescriptive, "isolated" line styles:
-        <ul>
-            <li>Navigation selection underline</li>
-            <li>Tab underline</li>
-            <li>Tile crown</li>
-            <li>Row separator</li>
-        </ul>
-    </li>
-</ul>
+  <div class="definition-table definition-table-wide">
+    <div class="definition-item">
+      <div class="swatch" style="border: 1px solid #cdcfd2; border-radius: 3px;"></div>
+    </div>
+    <div class="definition-props">
+      <div class="definition-usage">
+        <div class="bb-field-label">Usage</div>
+        <div>Rounded corners are primarily for interactive elements such as <a routerLink="../../components/button">buttons</a> to convey a distinction from non-interactive elements.</div>
+      </div>
+      <div class="definition-code">
+        <div class="bb-field-label">Code</div>
+        <code>border-radius: 3px</code>
+      </div>
+      <div class="definition-styling">
+        <div class="bb-field-label">Styling</div>
+        <div>border-radius: 3px</div>
+      </div>
+    </div>
+  </div>
+  
+<h2 class="bb-section-heading" id="shadows">Shadows</h2>
 
-<h2 class="bb-section-heading">Generic styles</h2>
-{{ include 'includes/design-table.hbs' items=itemsGeneric }}
+<p>In general, the use of shadows should be handled by the component. Specific interface designs should be flat aside from these components, using other techniques to create visual separation.</p>
```

Comment:
I'd change the first sentence to use active voice: "In general, the component should handle the use of shadows."

Also, I'd delete "Specific" from the start of the second sentence. And I'd change "using" to "and use" and delete the comma.